classes/04class/exercises/c04-iac01/README.md

```diff
@@ -0,0 +1,15 @@
+# S3 Bucket and EC2 Instance (c04-iac01)
+
+In this exercise, you'll need to an EC2 instance and an S3 Bucket. The EC2 instance needs to be accessible through SSH(Port 22) and the EC2 instance needs to have permission to access the S3 bucket being created in this exercise. All resources must be created through Terraform code, with the exception of the EC2 Keypair, that you can use an existing one from your AWS Account.
```

Comment:
mention here that this is based on the exercise done on class 1

Reply:
added.